sui-execution/latest/sui-move-natives/src/eth_state_proof.rs

```diff
@@ -191,12 +265,20 @@ pub(crate) fn verify_eth_state(
         optimistic_update,
     };
 
-    eth_state.verify_and_apply_updates(&updates).map_err(|e| {
-        error!("failed to verify updates: {:?}", e);
-        PartialVMError::new(StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR)
-    })?;
+    if (is_init) {
+        eth_state
+            .verify_and_apply_initial_updates(&updates)
```

Comment:
can you link the code for this function `verify_and_apply_initial_updates()`? I can't find it.

Reply:
This is probably because the helios PR we did is not merged yet.
You can find it here:
https://github.com/dwallet-labs/helios/pull/23/files